src/components/ImageLoader.vue

```diff
@@ -52,7 +52,7 @@
      },
      colormap: {
        type: String,
-       default: 'Viridis'
+       default: 'Cividis'
```

Comment:
@Ren22 you should double check with @andy-d-palmer if we want to use Cividis as the default colour map

Reply:
Ok, will do that @intsco

Reply:
@Ren22 please leave the default as Viridis

Reply:
@andy-d-palmer got it